clang/test/Driver/sycl-offload-wrapper-compression.cpp

```diff
@@ -11,9 +11,10 @@
 // RUN: %clangxx -### -fsycl --offload-compression-level=3 %s 2>&1 | FileCheck %s --check-prefix=CHECK-NO-COMPRESS
 
 // For SYCL offloading to HIP, make sure we don't pass '--compress' to offload-bundler.
-// Clang thows an error in the following command "cannot find 'remangled-l64-signed_char.libspirv-amdgcn-amd-amdhsa.bc'", that's why adding 'not'.
+// Clang throws an error in the following command "cannot find 'remangled-l64-signed_char.libspirv-amdgcn-amd-amdhsa.bc'" on some machine,
+// that's why appending '|| true'. This error should not affect the driver invocation, which this test intends to verify.
 
-// RUN: not %clangxx -### -fsycl --offload-compress -Xsycl-target-backend=amdgcn-amd-amdhsa --offload-arch=gfx1031 -fsycl-targets=amdgcn-amd-amdhsa %s > %t.driver 2>&1
+// RUN: (%clangxx -### -fsycl --offload-compress -Xsycl-target-backend=amdgcn-amd-amdhsa --offload-arch=gfx1031 -fsycl-targets=amdgcn-amd-amdhsa %s > %t.driver 2>&1) || true
```

Comment:
Instead of using `|| true` here, add the `-fno-sycl-libspirv -nogpulib` options to the command line to limit library requirements.